odoo/tools/speedscope.py

```diff
@@ -91,8 +91,11 @@ def add_default(self):
         for key, profile in self.profiles_raw.items():
             sql = profile and profile[0].get('query')
             if sql:
+                # constant time help to compare determinism by making each frame last exactly one second
+                # self.add_output([key], hide_gaps=True, constant_time=True, display_name=f'{key} (constant time)')
```

Comment:
???

Reply:
I will remove that but I would like to keep it one way or another. 
This output would be added if we have only sql, but in first position it is not a good idea, but it was convenient for me to spot the sources of indeterminism. I'll change that.

Here in my results, I can compare if two fonctions call have the same number of queries by looking at the execution time.